org/ystia/docker/images/playbooks/create.yaml

```
@@ -0,0 +1,68 @@
+#
+# Ystia Forge
+# Copyright (C) 2018 Bull S. A. S. - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+# Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+#
+- name: Install ansible docker_image requirements
+  hosts: all
+  become: true
+  tasks:
+  - name: Get python version
+    python_requirements_info:
+    register: pri
+    failed_when: pri == None or pri.python_version == None or pri.python_version == ''
+  - name: Get python major version
+    set_fact:
+      python_major_version: "{{pri.python_version | replace('\n', '') | regex_replace('^(\\d+).*', '\\1') }}"
+  - name: Set python version for pip
+    set_fact:
+      python_pip_pkg: "python3-pip"
+      pip_cmd: "pip3"
+    when: python_major_version != "2"
+  - name: Set python 2 version for pip
+    set_fact:
+      python_pip_pkg: "python-pip"
+      pip_cmd: "pip"
+    when: python_major_version == "2"
+  - name: RedHat - install prerequisites
+    yum:
+      name:
+        - "{{ python_pip_pkg }}"
+      state: present
+      update_cache: yes
+    when: ansible_os_family == 'RedHat'
+  - name: Debian - install prerequisites
+    apt:
+      name:
+        - "{{ python_pip_pkg }}"
+      state: present
+      update_cache: yes
+    when: ansible_os_family == 'Debian'
```

If you have 2 containers that runs in // apt will likely fail on one instance as long as the first one haven't finish it's install.

```suggestion
    register: apt_res
    retries: 5
    delay: 15
    until: apt_res is success
    when: ansible_os_family == 'Debian'
```

---

ok!